app/src/scenes/support-center/dashboard.js

```diff
@@ -85,6 +85,9 @@ export default () => {
 
   return (
     <HeroContainer>
+      <NavLink className="link" style={{ color: "#32257F", fontWeight: "bold" }} to="/profil"> <svg width="8" height="11" viewBox="0 0 6 9" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
le `/profil` n'existe pas, on peut rediriger vers la dashboard directement je pense
```suggestion
      <NavLink style={{ color: "#32257F", fontWeight: "bold" }} to="/"> <svg width="8" height="11" viewBox="0 0 6 9" fill="none" xmlns="http://www.w3.org/2000/svg">
```